staging/src/slime.io/slime/modules/meshregistry/pkg/source/nacos/httpclient.go

```diff
@@ -121,3 +152,48 @@ func (c *client) Instances() ([]*instanceResp, error) {
 	}
 	return instanceAll, nil
 }
+
+func (c *client) login() {
+	if c.username == "" || c.password == "" {
+		return
+	}
+	url := c.chooseURL() + loginPath
+	body := strings.NewReader("username=" + c.username + "&password=" + c.password)
+	resp, err := c.call(http.MethodPost, url, body)
```

Comment:
i'm not sure if server will revoke the previous token when we re-login and if that the current access may get 403 err.